src/views/blocks/tickets/footer-total.php

```diff
@@ -27,6 +33,6 @@
 		<?php echo esc_html_x( 'Total:', 'Total selected tickets price.', 'event-tickets' ); ?>
 	</span>
 	<span class="tribe-tickets__footer__total__wrap">
-		<?php echo $tribe_commerce_currency->get_formatted_currency_with_symbol( 0, $post_id, $provider->class_name ); ?>
+		<?php echo $tribe_commerce_currency->get_formatted_currency_with_symbol( 0, $post_id, $provider_class ); ?>
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$tribe_commerce_currency'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$tribe_commerce_currency'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$post_id'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$post_id'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$provider_class'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$provider_class'.

<sub>hash: c075af508885c6183a92ef80a914f6e1</sub>